<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qdialog_agregarusuarios</class>
 <widget class="QDialog" name="qdialog_agregarusuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_nombre">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_nombre"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_nivelusuario">
       <property name="text">
        <string>Nivel de Usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_nivelusuario">
       <item>
        <property name="text">
         <string>Nivel 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nivel 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nivel 2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_apellido">
       <property name="text">
        <string>Apellido:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_apellido"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_legajo">
       <property name="text">
        <string>Legajo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_legajo"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_pass_01">
       <property name="text">
        <string>Contraseña:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pass_01"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_pass_02">
       <property name="text">
        <string>Repetir Contraseña:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pass_02"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_agregarusuario">
     <property name="text">
      <string>Agregar Usuario</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_cancelar">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nombre</tabstop>
  <tabstop>lineEdit_apellido</tabstop>
  <tabstop>lineEdit_legajo</tabstop>
  <tabstop>comboBox_nivelusuario</tabstop>
  <tabstop>lineEdit_pass_01</tabstop>
  <tabstop>lineEdit_pass_02</tabstop>
  <tabstop>pushButton_agregarusuario</tabstop>
  <tabstop>pushButton_cancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
